woocommerce-myparcel.php

```diff
@@ -199,7 +198,7 @@ public function load_classes()
                 return;
             }
 
-            if (! $this->phpVersionMeets(self::MINIMUM_PHP_VERSION_5_4)) {
+            if (! $this->phpVersionMeets(self::PHP_VERSION_7_1)) {
```

Comment:
```php
const PHP_VERSION_7_1      = '7.1';

const PHP_VERSION_REQUIRED = self::PHP_VERSION_7_1;

// ...

if (! $this->phpVersionMeets(self::PHP_VERSION_REQUIRED)) {

// ...
$error = __("WooCommerce MyParcel requires PHP {PHP_VERSION} or higher.", "woocommerce-myparcel");
$error = str_replace('{PHP_VERSION}', self::PHP_VERSION_REQUIRED);
```
Of iets dergelijks... Geen php versie in de vertaling en je hoeft alleen nog maar de constante `PHP_VERSION_REQUIRED` aan te passen als hij een keertje verandert :)